src/main/java/uk/gov/companieshouse/extensions/api/config/HealthCheckController.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.extensions.api.config;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthCheckController {
+
+    @GetMapping("/extensions-api/healthcheck")
+    public ResponseEntity<String> healthcheck() {
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+}
```

Comment:
Not a big deal but going forwards could we take a look at using spring-boot-actuator for healthchecks if they aren't present. Saves on having to write out the classes